plutarch-test/conditional/Plutarch/TryFromSpec.hs

```diff
@@ -487,7 +489,9 @@ theField = unTermCont $ do
 
 ------------------- Sample usage DerivePNewType ------------------------------------
 
-newtype PWrapInt (s :: S) = PMkWrapInt (Term s PInteger)
-  deriving (PlutusType, PIsData, PEq, POrd) via (DerivePNewtype PWrapInt PInteger)
+newtype PWrapInt (s :: S) = PWrapInt (PInteger s)
+  deriving newtype (PIsData, PEq, POrd)
 
-deriving via DerivePNewtype (PAsData PWrapInt) (PAsData PInteger) instance PTryFrom PData (PAsData PWrapInt)
+instance PTryFrom PData (PAsData PWrapInt) where
+  type PTryFromExcess PData (PAsData PWrapInt) = PTryFromExcess PData (PAsData PInteger)
+  ptryFrom' t f = ptryFrom' t $ \(t', exc) -> f (coerce (t' :: Term _ (PAsData PInteger)), exc)
```

Comment:
Why do we not want deriving via `DerivePNewtype` anymore?

Reply:
It didn't work and was in fact illegal.

Reply:
I think we should rather just add a warning that this leads to issues if we instantiate DerivePNewtype incorrectly, I think that this instance is really helpful in practise 🤷